pkg/collect/logs_test.go

```diff
@@ -7,13 +7,13 @@ import (
 
 	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
 	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	testclient "k8s.io/client-go/kubernetes/fake"
 )
 
 func Test_setLogLimits(t *testing.T) {
+	maxBytes := int64(5000000)
```

Comment:
You can change this to a non default value (e.g 200) to test that case instead

Reply:
I should have documented that in the PR comment, thanks for highlighting. I'll make a small change.